arch/xtensa/core/userspace.S

```diff
@@ -265,8 +265,6 @@ xtensa_userspace_enter:
 	/* stash user stack */
 	l32i a0, a1, 4
 
-	addi a1, a1, 28
-
```

Comment:
Should we maybe audit where this came from to make sure a step didn't get skipped somewhere?  I guess it's probably fine since whatever 28-byte block on the stack had been there is being forgotten anwyay.

Reply:
The space was used to stash incoming arguments (starting from line 230) which we don't care anymore from this point on.